[Speex] Build failure on mac osx

**Host Environment**
- OS: mac osx 11 (github actions VM)
- Compiler: revision

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install speex:x64-osx` or `./vcpkg install speex:arm64-osx` 
(as long as automake/autoconf/libtool are not already installed on your machine)

**Failure logs** 
Building package speex[core]:arm64-osx...
-- Using community triplet arm64-osx. This triplet configuration is not guaranteed to succeed.
-- [COMMUNITY] Loading triplet configuration from: /usr/local/share/vcpkg/triplets/community/arm64-osx.cmake
-- Installing port from location: /usr/local/share/vcpkg/buildtrees/versioning_/versions/speex/0f443cef71fd44439aa55[90](https://github.com/Kolfering/alephone/runs/5503023836?check_suite_focus=true#step:3:90)63a219da440f6665d
-- Downloading https://github.com/xiph/speex/archive/Speex-1.2.0.tar.gz -> xiph-speex-Speex-1.2.0.tar.gz...
-- Extracting source /usr/local/share/vcpkg/downloads/xiph-speex-Speex-1.2.0.tar.gz
-- Using source at /usr/local/share/vcpkg/buildtrees/speex/src/peex-1.2.0-b5745efd03.clean
-- Getting CMake variables for arm64-osx-dbg
-- Getting CMake variables for arm64-osx-rel
-- Generating configure for arm64-osx
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:128 (message):
    Command failed: /usr/local/bin/autoreconf -vfi
    Working Directory: /usr/local/share/vcpkg/buildtrees/speex/src/peex-1.2.0-b5745efd03.clean/
    Error code: 2
    See logs for more information:
      /usr/local/share/vcpkg/buildtrees/speex/autoconf-arm64-osx-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:625 (vcpkg_execute_required_process)
  buildtrees/versioning_/versions/speex/0f443cef71fd44439aa559063a219da440f6665d/portfile.cmake:30 (vcpkg_configure_make)
  scripts/ports.cmake:145 (include)


Error: Building package speex:arm64-osx failed with: BUILD_FAILED

**Additional context**

Looks like some tools are needed to build this package but vcpkg doesn't install them automatically.
Autoconf and libtool are already installed when it fails, I have to install Automake too to make the speex build succeed.

--- a/ports/speex/portfile.cmake
+++ b/ports/speex/portfile.cmake
@@ -28,7 +28,9 @@ if(VCPKG_TARGET_IS_WINDOWS)
   endif()
 else()
   if(VCPKG_TARGET_IS_OSX)
-      message("${PORT} currently requires the following library from the system package manager:\n    autoconf\n    automake\n    libtool\n\nIt can be installed with brew install autoconf automake libtool")
+      message("${PORT} currently requires the following libraries from the system package manager:\n    autoconf\n    automake\n    libtool\n\nIt can be installed with brew install autoconf automake libtool")
+  elseif(VCPKG_TARGET_IS_LINUX)
+      message("${PORT} currently requires the following libraries from the system package manager:\n    autoconf\n    automake\n    libtool\n\nIt can be installed with apt-get install autoconf automake libtool")
   endif()
   vcpkg_configure_make(
     SOURCE_PATH ${SOURCE_PATH}

--- a/ports/speex/portfile.cmake
+++ b/ports/speex/portfile.cmake
@@ -27,6 +27,9 @@ if(VCPKG_TARGET_IS_WINDOWS)
     )
   endif()
 else()
+  if(VCPKG_TARGET_IS_OSX)
+      message("${PORT} currently requires the following library from the system package manager:\n    autoconf\n    automake\n    libtool\n\nIt can be installed with brew install autoconf automake libtool")
+  endif()
   vcpkg_configure_make(
     SOURCE_PATH ${SOURCE_PATH}
     AUTOCONFIG

--- a/ports/speex/vcpkg.json
+++ b/ports/speex/vcpkg.json
@@ -1,9 +1,10 @@
 {
   "name": "speex",
   "version": "1.2.0",
-  "port-version": 10,
+  "port-version": 11,
   "description": "Speex is an Open Source/Free Software patent-free audio compression format designed for speech.",
   "homepage": "https://github.com/xiph/speex",
+  "license": null,
   "dependencies": [
     {
       "name": "vcpkg-cmake",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6598,7 +6598,7 @@
     },
     "speex": {
       "baseline": "1.2.0",
-      "port-version": 10
+      "port-version": 11
     },
     "speexdsp": {
       "baseline": "1.2.0",

--- a/versions/s-/speex.json
+++ b/versions/s-/speex.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "54f71edc9b32fe89de70547ef738c7ae72d48d43",
+      "version": "1.2.0",
+      "port-version": 11
+    },
+    {
       "git-tree": "0f443cef71fd44439aa559063a219da440f6665d",
       "version": "1.2.0",
       "port-version": 10

--- a/versions/s-/speex.json
+++ b/versions/s-/speex.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "54f71edc9b32fe89de70547ef738c7ae72d48d43",
+      "git-tree": "7cbe163c0a5fabd08cb6665c092d6b13dfa51d69",
       "version": "1.2.0",
       "port-version": 11
     },

